go/libkb/secret_store.go

```diff
@@ -125,24 +125,36 @@ func NewSecretStoreLocked(g *GlobalContext) *SecretStoreLocked {
 }
 
 func (s *SecretStoreLocked) RetrieveSecret(username NormalizedUsername) ([]byte, error) {
+	if s == nil || s.SecretStoreAll == nil {
+		return nil, nil
+	}
 	s.Lock()
 	defer s.Unlock()
 	return s.SecretStoreAll.RetrieveSecret(username)
 }
 
 func (s *SecretStoreLocked) StoreSecret(username NormalizedUsername, secret []byte) error {
+	if s == nil || s.SecretStoreAll == nil {
+		return nil
```

Comment:
This is all temporary...once the SecretStoreFile branch makes it to master, it will never be nil.